examples/quickstart/quickstart.ipynb

```diff
@@ -407,7 +407,7 @@
     "# Initialise a new pipeline\n",
     "second_pipeline = mnist_pipeline(\n",
     "    importer=importer_fashion_mnist(),\n",
-    "    trainer=trainer(config=TrainerConfig(epochs=1)),\n",
+    "    trainer=trainer(),\n",
```

Comment:
Isn't it cooler to add the config to show it off?

Reply:
I get the following error when I run that:

```
---------------------------------------------------------------------------
StepInterfaceError                        Traceback (most recent call last)
<ipython-input-23-8e51e730bb32> in <module>()
      2 first_pipeline = mnist_pipeline(
      3     importer=importer(),
----> 4     trainer=trainer(config=TrainerConfig(epochs=1)),
      5     evaluator=evaluator(),
      6 )

/usr/local/lib/python3.7/dist-packages/zenml/steps/base_step.py in __init__(self, *args, **kwargs)
    164         if len(kwargs) > maximum_kwarg_count:
    165             raise StepInterfaceError(
--> 166                 f"Too many keyword arguments ({len(kwargs)}, "
    167                 f"expected: {maximum_kwarg_count}) passed when "
    168                 f"creating a '{self.step_name}' step."

StepInterfaceError: Too many keyword arguments (1, expected: 0) passed when creating a 'trainer' step.
```

I didn't dig into exactly what this was connected to, so thought I'd just remove for now. Is there a quick fix that you can see?